hosters/hetzner-dedicated/hetzner-dedicated-wipe-and-install-nixos.sh

```diff
@@ -92,13 +92,23 @@ done
 [[ -n $VERBOSE ]] && set -x
 
 # Install dependencies
-apt-get update
 apt-get install -y sudo
 
 # Inspect existing disks
 declare -a DISKS
 readarray -t DISKS < <(lsblk -Jl | jq -r '.blockdevices[] | select(.type == "disk") | .name')
 
+# Helper function to get the correct partition name given a disk name and partition number
+# partname sda 2 => /dev/sda2
+# partname nvme0p1 2 => /dev/nvme0p1n2
```

Comment:
The functionality is correct, though this example is wrong in 2 ways, first `nvme0p1` isn't right, second it would produce `/dev/nvme0p1p2`, it should be `partname nvme0n1 2 => /dev/nvme0n1p2`

Reply:
Indeed, I fixed the function but forgot to fix the comment. Fixed now.